examples/web/assets/js/app.js

```diff
@@ -53,14 +53,21 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	desktopJS.Container.addListener("layout-loaded", (e) => console.log("Layout loaded - static: " + e.layoutName));
 
 	subscribe();
+
+	// If url is provided a hash, try to navigate to bootstrap tab if exists
+	var activeTab = $('[href="' + location.hash + '"]');
+	activeTab && activeTab.tab('show');
+
+	// Enable popovers
+	$('[data-toggle="popover"]').popover();
 });
 
 openWindowButton.onclick = function () {
 	childWindow = container.createWindow("http://localhost:8000",
 		{
 			resizable: true,
 			x: 10, y: 10,
-			width: 500, height: 500,
+			width: 850, height: 500,
```

Comment:
Bootstrap dynamic grid columns are a bit annoying for larger displays with regards to wrapping so make child windows wider to avoid wrapping on initial display.